docs-src/references/cluster-metrics.md

```diff
@@ -0,0 +1,177 @@
+Example: _need an example_
+
+The `operation` tag contains details about Task type and Active vs Standby statuses, and can be used to get request rates, error rates or latencies per operation, which can help identify issues caused by database problems.
+
+## Persistence metrics
+
+Temporal Server emits metrics for every persistence database read and write.
+Some of the most important metrics are:
+
+### [`persistence_requests`](https://github.com/temporalio/temporal/blob/57424088d28e785d4dfe797c4a957f093e5ad02e/common/metrics/defs.go#L2523)
+
+Description: Emit on every persistence request.
+Examples:
+
+- Prometheus query for getting total number of persistence requests by operation for the history service:
+  `sum by (operation) (rate(persistence_requests{service="$service",service_name="history"}[1m]))`
+- Prometheus query for getting total number of persistence requests by operation for the matching service:
+  `sum by (operation) (rate(persistence_requests{cluster="$cluster",service_name="matching"}[5m]))`
+
+### [`persistence_errors`](https://github.com/temporalio/temporal/blob/57424088d28e785d4dfe797c4a957f093e5ad02e/common/metrics/defs.go#L2524)
+
+Description: Emit on persistence error, includes `error_type` tag. This metric is a good indicator for connection issues between Server and the Persistence store.
+Example:
+
+- Prometheus query for getting the persistence errors by service:
+  `sum(rate(persistence_error_with_type{service="$service",service_name="history"}[1m])) by (error_type)`
+
+### [`persistence_latency`](https://github.com/temporalio/temporal/blob/57424088d28e785d4dfe797c4a957f093e5ad02e/common/metrics/defs.go#L2526):
+
+Description: _need description here_.
+Example:
+
+- Prometheus query for getting latency by percentile:
+  `histogram_quantile($percentile, sum(rate(persistence_latency_bucket{service="$service" service_name="history"}[1m])) by (operation, le))`
+
+## Relevant SDK Metrics for Workflows and Activities
+
+- To monitor task queue backlog look at the `schedule_to_start` latency metrics, for example, `temporal_workflow_task_schedule_to_start_latency_seconds_bucket` and `temporal_activity_schedule_to_start_latency_seconds_bucket` metrics.
+- For Activity Workers, use the `temporal_worker_task_slots_available` SDK metric with label `worker_type="ActivityWorker"` to help you tune your `WorkerOptions#getMaxConcurrentActivityExecutionSize` setting.
+- `Workflow_endtoend_latency` - measures your Workflow Execution time from start to close.
+- Temporal emits counters on completion of Workflows. These are useful in getting the overall statistics about Workflow completion. Use `workflow_success`, `workflow_failed`, `workflow_timeout`, `workflow_terminate` and `workflow_cancel` counters for each type of Workflow completion. These metrics are also tagged with `namespace` tag.
+
+## Cluster performance baselines
```

Comment:
I think @yiminc suggested we drop this particular bit for now.

Obviously we want to address performance baselines, but this feels wildly incomplete and thus misleading.

So my preference would be to drop this section for now - and then immediately follow up on this as a KB article.